app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -19,36 +22,64 @@ export default function NewNoteForm({ onClose, refetch }) {
   };
   return (
     <Formik
-      initialValues={{
-        title: "",
-        description: ""
-      }}
+      initialValues={FORM_INITIAL_VALUES.addNoteForm}
       onSubmit={handleSubmit}
-      validationSchema={yup.object({
-        title: yup.string().required("Title is required"),
-        description: yup.string().required("Description is required")
-      })}
+      validationSchema={FORM_VALIDATION_SCHEMAS.addNoteForm}
     >
-      {({ isSubmitting }) => (
+      {({ isSubmitting, handleChange }) => (
         <Form>
-          <Input label="Title" name="title" className="mb-6" />
-          <Textarea label="Description" name="description" rows={8} />
-          <div className="nui-pane__footer nui-pane__footer--absolute">
-            <Button
-              onClick={onClose}
-              label="Cancel"
-              size="large"
-              style="secondary"
+          <div className="px-10">
+            <Input
+              label="Title"
+              name="title"
+              placeholder="Enter note title"
+              className="mb-6"
+              required
+            />
+            <Input
+              label="Description"
+              name="description"
+              placeholder="Enter note description"
+              className="mb-6"
+              required
+            />
+            <Select
+              label="Assigned Contact"
+              name="assigned-contact"
+              options={FORM_SELECT_OPTIONS.addNoteForm.assignedContact}
+              onChange={({ value }) => handleChange("assigned-contact")(value)}
+              placeholder="Select a Contact"
+              className="mb-6"
+              required
+            />
+            <Select
+              label="Tags"
+              name="tags"
+              options={FORM_SELECT_OPTIONS.addNoteForm.tags}
+              onChange={({ value }) => handleChange("tags")(value)}
+              placeholder="Select a Tag"
+              className="mb-6"
+              required
             />
+          </div>
```

Comment:
Rather than adding `mb-6` to each of the individual blocks, it's better to use `space-y-*` utils from tailwindCSS.